tests/initialisation.py.exp

```diff
@@ -0,0 +1,188 @@
+       [0.0, 1.0, 0.0, 0.0],
+       [0.0, 0.0, 1.0, 0.0]], dtype=float64)
+array([[0.0, 0.0, 0.0, 0.0],
+       [1.0, 0.0, 0.0, 0.0],
+       [0.0, 1.0, 0.0, 0.0]], dtype=float64)
+array([[0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0],
+       [1.0, 0.0, 0.0, 0.0]], dtype=float64)
+array([[0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0]], dtype=float64)
+array([[0.0, 1.0, 0.0, 0.0],
+       [0.0, 0.0, 1.0, 0.0],
+       [0.0, 0.0, 0.0, 1.0]], dtype=float64)
+array([[0.0, 0.0, 1.0, 0.0],
+       [0.0, 0.0, 0.0, 1.0],
+       [0.0, 0.0, 0.0, 0.0]], dtype=float64)
+array([[0.0, 0.0, 0.0, 1.0],
+       [0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0]], dtype=float64)
+Array creation using FULL:
+array([[3.0, 3.0, 3.0, 3.0],
+       [3.0, 3.0, 3.0, 3.0]], dtype=float64)
+array([[3, 3, 3, 3],
+       [3, 3, 3, 3]], dtype=uint8)
+Array creation using LINSPACE:
+array([0.0, 2.5, 5.0, 7.5, 10.0], dtype=float64)
+array([0.0, 2.0, 4.0, 6.0, 8.0], dtype=float64)
+array([0.0, 2.5, 5.0, 7.5, 10.0], dtype=float64)
+array([0, 2, 4, 6, 8], dtype=uint8)
+array([0, 2, 4, 6, 8], dtype=uint16)
+array([0, 2, 4, 6, 8], dtype=int8)
+array([0, 2, 4, 6, 8], dtype=int16)
+Array creation using LOGSPACE:
+array([1.0, 316.227766016838, 100000.0, 31622776.6016838, 10000000000.0], dtype=float64)
+array([1.0, 100.0, 10000.0, 1000000.0, 100000000.0], dtype=float64)
+array([1.0, 316.227766016838, 100000.0, 31622776.6016838, 10000000000.0], dtype=float64)
+array([1, 100, 16, 64, 0], dtype=uint8)
+array([1, 100, 10000, 16960, 57600], dtype=uint16)
+array([1, 100, 16, 64, 0], dtype=int8)
+array([1, 100, 10000, 16960, -7936], dtype=int16)
+Array creation using ZEROS:
+array([0.0, 0.0, 0.0], dtype=float64)
+array([[0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0]], dtype=float64)
+array([[0, 0, 0],
+       [0, 0, 0],
+       [0, 0, 0]], dtype=uint8)
+array([[0, 0, 0],
+       [0, 0, 0],
+       [0, 0, 0]], dtype=uint16)
+array([[0, 0, 0],
+       [0, 0, 0],
+       [0, 0, 0]], dtype=int8)
+array([[0, 0, 0],
+       [0, 0, 0],
+       [0, 0, 0]], dtype=int16)
+array([[0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0]], dtype=float64)
+array([[0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0],
+       [0.0, 0.0, 0.0, 0.0]], dtype=float64)
+Array creation using ONES:
+array([1.0, 1.0, 1.0], dtype=float64)
+array([[1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0]], dtype=float64)
+array([[1, 1, 1],
+       [1, 1, 1],
+       [1, 1, 1]], dtype=uint8)
+array([[1, 1, 1],
+       [1, 1, 1],
+       [1, 1, 1]], dtype=uint16)
+array([[1, 1, 1],
+       [1, 1, 1],
+       [1, 1, 1]], dtype=int8)
+array([[1, 1, 1],
+       [1, 1, 1],
+       [1, 1, 1]], dtype=int16)
+array([[1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0]], dtype=float64)
+array([[1.0, 1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0, 1.0],
+       [1.0, 1.0, 1.0, 1.0]], dtype=float64)
```

Comment:
Perhaps, the newline character is missing here, too.

Reply:
Yeah, I noticed that too.